test/testServer.js

```diff
@@ -1,15 +1,22 @@
+var istanbulMiddleware = require('istanbul-middleware');
```

Comment:
Selenium isn't an amazing fit for unit testing but I chose it to gain familiarity with the aim that we could use it for automated regression testing. That being said to turn it into a unit tester this .js boots up a mini server to stand in RB & Yolanda's place as well as kicking off the selenium test and reporting its results.